setup.py

```diff
@@ -123,26 +128,22 @@ def run(self):
                 )
 
                 try:
+           
                     if not os.path.isdir(os.path.join(LRMQDIST(), '.git')):
-                        print('- pull submodule rabbitmq-c...')
-                        if os.path.isfile('Makefile'):
-                            os.system(' '.join([make, 'submodules']))
-                        else:
-                            os.system(' '.join(['git', 'clone', '-b', 'v0.8.0', 
-                                'https://github.com/alanxz/rabbitmq-c.git',
-                                'rabbitmq-c']))
+                         print('- pull submodule rabbitmq-c...')
+                         if os.path.isfile('Makefile'):
+                             os.system(' '.join([make, 'submodules']))
+                         else:
+                             os.system(' '.join(['git', 'clone', '-b', 'master',
+                                 'https://github.com/alanxz/rabbitmq-c.git',
+                                 'rabbitmq-c']))
 
                     os.chdir(LRMQDIST())
 
-                    if not os.path.isfile('configure'):
-                        print('- autoreconf')
-                        os.system('autoreconf -i')
-
-                    if not os.path.isfile('config.h'):
-                        print('- configure rabbitmq-c...')
-                        if os.system('/bin/sh configure --disable-tools \
-                                --disable-docs --disable-dependency-tracking'):
-                            return
+                    print('- cmake')
+                    os.system('cmake .')
```

Comment:
Maybe we should use PEP 517 instead?

Reply:
could you point me to a sample of this being used ?

Reply:
@thedrow, would it involve creating a static wheel with openssl binary in it, since this PR enables SSL support.